src/main/java/ch/njol/skript/effects/EffMessage.java

```diff
@@ -37,20 +40,34 @@
 import ch.njol.skript.util.chat.BungeeConverter;
 import ch.njol.skript.util.chat.ChatMessages;
 import ch.njol.util.Kleenean;
+import net.md_5.bungee.api.chat.BaseComponent;
+import net.md_5.bungee.api.chat.TextComponent;
 
 @Name("Message")
 @Description({"Sends a message to the given player. Only styles written",
-		"in given string or in <a href=expressions.html#ExprColoured>formatted expressions</a> will be parsed."})
+		"in given string or in <a href=expressions.html#ExprColoured>formatted expressions</a> will be parsed.",
+		"Adding an optional sender allows the messages to be sent as if a specific player sent them.",
+		"This is useful with Minecraft 1.16.4's new chat ignore system."})
```

Comment:
The chat blocking system doesn't work if you don't send it as a player, right? If so, maybe it'd be better to say something like: `This allows the chat ignore system added in Minecraft 1.16.4 to block these messages`

Up for discussion :D

Reply:
Yes, Good call. Ill all a little more to the description.

Reply:
done 👍